### ISSUE TITLE
Feature request: "where" query support when getting rows from a view

### ISSUE THREAD

**Issue by mwvaughn** [TACC/paas#37]
Views act as virtual tables. Tables can be queried using a "where" parameter. Thus, views should also be queryable using a where parameter.

**Comment by NotChristianGarcia** [comment]
I'm ruminating on this. The issue is "where" for tables isn't even particularly safe. ~~As in, at all. It might actually be wise to get rid of "where" for tables.~~

~~We can have user sql for creating views because that's behind an admin role check. But "where" for tables and views are public and there's no easy way I see to bypass sql injection. I'll look into third-party packages to see if there's anything that can be done. Will stay on the backburner a bit longer.~~

~~We can put "where" queries behind another admin check, but then users won't have access, this might be a decent compromise though.~~

We can make "where" queries for tables very safe. "Where" for views will require a two query query though, first to get all columns, and then to execute the actual command. We have to use parameterized queries, but for that to be safe, we must know all columns for the object. We save table columns, and we can get columns for a view in real time.

tldr: It's poossible, views query will not be fast, this will be a decent amount of work as we probably need to implement the search spec at the same time. Currently on the backburner as I go through the other issues first.
